app/src/main/java/com/keylesspalace/tusky/viewmodel/EditProfileViewModel.kt

```diff
@@ -51,6 +50,16 @@ import javax.inject.Inject
 private const val HEADER_FILE_NAME = "header.png"
 private const val AVATAR_FILE_NAME = "avatar.png"
 
+/**
+ * Conveniently groups Profile Data users can modify in the UI.
```

Comment:
`ProfileDataInUi` ?

Or simply ProfileData - because the other data class (the diff) has a more special name and can be differentiated.

Then the comment can be removed.

(Code should always speak for itself, if it can.)